README.md

```diff
@@ -168,6 +168,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Abhisman Sarkar](https://github.com/abs007)
   - [Abiola Farounbi](https://github.com/Abiola-Farounbi)
   - [Abuzer Asif](https://github.com/abuzerasif)
+  - [Abubakar sadiq](https://github.com/ABuabakarsadiqq)
```

Comment:
Please add your name in line 170, after Abiola Farounbi. Also, it'd be better if you capitalize the `s` in your last name. Do let us know if you need any help with anything! 🙂

Reply:
@Panquesito7 I've added and pushed the changes boss🙂, please do I need to create another pull request ??

Reply:
> @Panquesito7 I've added and pushed the changes boss🙂, please do I need to create another pull request ??

Great 👍, Now you don't have to create another PR.